src/js/actions/libraries.js

```diff
@@ -118,6 +118,17 @@ define(function (require, exports) {
     ];
     
     /**
+     * List of graphic representations types that are editable in DS.
+     * Due to the limitation in DS, we only support editing psd file.
+     * 
+     * TODO we should support editing images (png, jpg, etc.) by running the 
+     * flatten command.
+     */
+    var EDITABLE_GRAPHIC_REPRESENTATION_TYPES = [
```

Comment:
It looks like this is only used as a set. Would it make more sense for it to be a `window.Set` object?